readme.md

```diff
@@ -0,0 +1,148 @@
+# Forms
+
+A set of utilities to help you create forms in React.
+
+## Motivation
+
+Creating ad-hoc forms using React hooks is quite easy, but there is one thing that is a bit hard to
+do: preventing to render the complete form on each keystroke.
+
+Another motivation is to make form handling within our applications more consistent.
+
+## Installation
+
+```
+yarn add @kaliber/forms
+```
+
+## Usage
+
+Please look at the example for more advanced use-cases.
+
+```jsx
+import { useForm, useFormField, required, email } from '@kaliber/forms'
+
+const validationErrors = {
+  required: 'This field is required',
+  email: 'This is not a valid e-mail',
+}
+
+export function Simple() {
+  const { form: { fields }, submit } = useForm({
+    initialValues: {
+      name: '',
+      email: '',
+    },
+    fields: {
+      name: required,
```

Comment:
Ik mis een stukje uitleg over wat het precies is dat je hier meegeeft. Een validatie functie was niet hetgeen waar ik aan dacht (achteraf is het wel logisch).

Reply:
Goed punt, zou al duidelijk in de readme moeten staan denk ik